frontend/src/scenes/session-recordings/player/sessionRecordingPlayerLogic.ts

```diff
@@ -372,91 +415,75 @@ export const sessionRecordingPlayerLogic = kea<sessionRecordingPlayerLogicType>(
         ],
         playingTimeTracking: [
             {
-                isPlaying: false as boolean,
-                isBuffering: false as boolean,
-                lastTimestamp: null as number | null,
+                state: 'unknown',
+                lastTimestamp: null,
                 watchTime: 0,
                 bufferTime: 0,
-            },
+            } as PlayerTimeTracking,
             {
+                updatePlayerTimeTracking: (state) => {
+                    // called on a timer to avoid inactive watching from not capturing a clear time
+                    return ['playing', 'buffering'].includes(state.state)
+                        ? updatePlayerTimeTracking(state, state.state)
+                        : state
+                },
```

Comment:
if you're just watching the screen and then click another recording we didn't have an action to react to update this value... so let's update it on a timer